SPECS/python-fasteners/python-fasteners.spec

```diff
@@ -0,0 +1,196 @@
+# Sphinx-generated HTML documentation is not suitable for packaging; see
+# https://bugzilla.redhat.com/show_bug.cgi?id=2006555 for discussion.
+#
+# We can generate PDF documentation as a substitute.
+# MARINER - Not creating doc rpm package.
+
+# The python-diskcache package, used in some of the tests, has been retired.
+%bcond_with diskcache
+
+Name:           python-fasteners
+Version:        0.17.3
+Release:        3%{?dist}
+Summary:        A python package that provides useful locks
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/harlowja/fasteners
+# We need to use the GitHub archive instead of the PyPI sdist to get tests.
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/fasteners-%{version}.tar.gz
+
+# “Remove futures from the requirements-test.txt”
+# Backport upstream commit 49d8f5b to remove a test dependency that is no
+# longer used and only supports Python 2.
+Patch:          %{url}/commit/49d8f5bb56157a82ff3e6128b506638a214e6d43.patch
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python-wheel
+%if %{with_check}
+BuildRequires:  python3-pip
+BuildRequires:  python3-pytest
+%endif
+
+%global common_description %{expand: \
+Cross platform locks for threads and processes}
+
+%description
+%{common_description}
+
+%package -n python3-fasteners
+Summary:        A python package that provides useful locks
+
+%description -n python3-fasteners
+%{common_description}
+
+%prep
+%autosetup -p1 -n fasteners-%{version}
+%if %{without diskcache}
+sed -r -i '/\b(diskcache)\b/d' requirements-test.txt
+%endif
+%generate_buildrequires
+%pyproject_buildrequires -r
```

Comment:
Please remove these two lines- not supported in Mariner

Reply:
Done. Added dependencies for runtime.